### PR TITLE
[NWO] Create community.amazon collection

### PR DESCRIPTION
**Pull request by jillr**
AWS modules will live in a single collection, distinct from community.general.
I don't anticipate any changes from this, but WIP while still testing just to be sure.

### CODE REVIEW

**Comment by jctanner** [comment]
LGTM

**Comment by jillr** [comment]
DNM: There's 2 mod_utils that are currently in _base that may move however unable to test, AWS `ansible_collections.community.amazon.plugins.module_utils.ec2.__spec__` is broken with NWO using last published ansible-base and there's a known issue with migrate.py preventing testing that version of "ansible-base" locally.
```
index 034cf17..96799a4 100644
--- a/scenarios/nwo/ansible.yml
+++ b/scenarios/nwo/ansible.yml
@@ -82,8 +82,6 @@ _core:
   - _text.py
   - ansible_release.py
   - api.py
-  - aws/core.py
-  - aws/urls.py
   - basic.py
   - common/_collections_compat.py
   - common/_json_compat.py
diff --git a/scenarios/nwo/community.yml b/scenarios/nwo/community.yml
index ee465b8..3454f7c 100644
--- a/scenarios/nwo/community.yml
+++ b/scenarios/nwo/community.yml
@@ -23,6 +23,8 @@ amazon:
   - aws/acm.py
   - aws/batch.py
   - aws/cloudfront_facts.py
+  - aws/core.py
+  - aws/urls.py
   - aws/direct_connect.py
   - aws/elb_utils.py
   - aws/elbv2.py
```

**Comment by jillr** [comment]
@jctanner I believe this is ready to merge.  Any further adjustments should be minor.

**Comment by jctanner** [comment]
@jillr can you rebase please?

File: scenarios/nwo/community.yml [hunk 1]
```diff
@@ -1,10 +1,300 @@
...
+  - aws/elb_utils.py
+  - aws/elbv2.py
+  - aws/iam.py
+  - aws/rds.py
+  - aws/s3.py
+  - aws/urls.py
+  - aws/waf.py
+  - aws/waiters.py
+  modules:
+  - cloud/amazon/_aws_acm_facts.py
```
Comment (pabelanger) [inline scenarios/nwo/community.yml]: You can also use globs:
  - cloud/amazon/*

Assuming everything under that folder is coming into this collection

File: scenarios/nwo/community.yml [hunk 2]
```diff
@@ -1,10 +1,300 @@
...
+  - cloud/amazon/_aws_kms_facts.py
+  - cloud/amazon/_aws_region_facts.py
+  - cloud/amazon/_aws_s3_bucket_facts.py
+  - cloud/amazon/_aws_sgw_facts.py
+  - cloud/amazon/_aws_waf_facts.py
+  - cloud/amazon/_cloudformation_facts.py
+  - cloud/amazon/_cloudfront_facts.py
+  - cloud/amazon/_cloudwatchlogs_log_group_facts.py
+  - cloud/amazon/_ec2_ami_facts.py
+  - cloud/amazon/_ec2_ami_find.py
```
Comment (pabelanger) [inline scenarios/nwo/community.yml]: This is a symlink, and can be delete. It resoves to ec2_asg_info.py

Comment (pabelanger) [reply scenarios/nwo/community.yml]: At least, I think it can be deleted. I am trying to confirm now

File: scenarios/nwo/community.yml (hunk 1 above)
Comment (jillr) [reply scenarios/nwo/community.yml]: Until netapp removes their modules from cloud/amazon in devel we have to specify.

**Pull request merged**